rhoas-client/src/test/java/com/redhat/service/bridge/rhoas/RhoasAPI.java

```diff
@@ -29,8 +30,25 @@
 
     @POST
     @Path("topic")
-    public TopicAndServiceAccountResponse createTopicAndConsumerServiceAccount(TopicAndServiceAccountRequest request) {
-        return rhoasClient.get().createTopicAndConsumerServiceAccount(request).await().atMost(Duration.ofSeconds(60));
+    public Topic createTopicAndConsumerServiceAccount(Request request) {
```

Comment:
Can you clarify why we need this api in the `test` project? I see 
```
/**
 * This dummy test API simulates a real usage of the {@link RhoasClient} in a real JAX-RS bean.
 * It allows {@link RhoasAPITest} to spot errors caused by the client mistakenly blocking the
 * Vert.x event loop thread, which can't be spotted testing directly the client in a QuarkusTest
 * (because blocking the test's main thread doesn't cause any error: it simply blocks).
 */
 ```
 but I do not see any test using such exception

Reply:
This API class is used by `RhoasAPITest` and was introduced in #196. Its purpose is to spot if the `RhoasClient` blocks the Vert.x event loop thread. There's no explicit assert on any explicit exception, the test would simply fail if a block occurs.

Does this answer your question?

Reply:
Uhm I'm not sure I get the point. Is it related to https://github.com/smallrye/smallrye-mutiny/issues/821 ?

Reply:
You're actually right: this API and the corresponding test are useless implemented this way.

My original idea was to implement the API using `quarkus-resteasy-reactive` and return `Uni<Topic>`, so that if, by mistake, a blocking piece of code went into the `RhoasClient` method, the test would fail with the following exception:
```
java.lang.IllegalStateException: The current thread cannot be blocked: vert.x-eventloop-thread-XYZ
```
These exceptions don't happen in normal tests because the main test thread can be blocked without issues.

Probably I changed the implementation when I realized that currently we never use `quarkus-resteasy-reactive` in our APIs and I didn't realize the test now lost its original purpose.

@danielezonca @r00ta @robpblake should I remove this? Or reimplement it with `quarkus-resteasy-reactive`?

Reply:
@kostola I'm fine to remove them if we are already covering the methods in other tests

Reply:
@kostola Fine for me to remove it, particularly when you say

> this API and the corresponding test are useless implemented this way.

I would hope that as we start to expand our integration tests, we would pick up issues in our clients that interact with external services rather than having to test this sort of functionality each time we write something reactive.

How do you feel about that?

Reply:
Removed :+1: